stdlib/public/Concurrency/Actor.swift

```diff
@@ -38,3 +38,21 @@ public func _defaultActorDestroy(_ actor: AnyObject)
 @_silgen_name("swift_defaultActor_enqueue")
 public func _defaultActorEnqueue(partialTask: PartialAsyncTask,
                                  actor: AnyObject)
+
+
+/// A singleton actor whose executor is equivalent to 
+/// \c DispatchQueue.main, which is the main dispatch queue.
+@globalActor public final class MainActor {
+  public static let shared = _MainActorImpl()
+  
+  public actor class _MainActorImpl {
+    public static func enqueue(partialTask: PartialAsyncTask) {
+      // TODO: to complete rdar://72105129, 
+      // figure out how to `import Dispatch` so this works:
+      // DispatchQueue.main.async { partialTask.run() }
```

Comment:
It might be best to implement an entry point in C++ to dispatch on the main actor, which will be a wrapper over Dispatch. That'll make it easier to replace Dispatch with something else later, and ensure that our Dispatch dependency is C-only.

Reply:
✅ done